### PR TITLE
Allow teachers to review student code

### PR DESCRIPTION
**Pull request by differentmatt**
Putting student code and solutions in student profile pages linked off
teacher class view.

### CODE REVIEW

File: app/templates/teachers/teacher-student-view.jade [hunk 1]
```diff
@@ -164,65 +164,78 @@ block content
...
-        //-       - if (!(instance && instance.hasMember(view.user)))
-        //-           div.small-details
-        //-             span=course.getTranslatedName()
-                
-        //- else
-        //-   p Apply a license to this student to assign additional courses!
-        //-   button.enroll-student-button.btn.btn-navy(data-i18n="teacher.apply_license", data-user-id=view.user.id, data-event-action="Teachers Class Enrollment Enroll Student")
-        //-   // this button doesn't work yet
-        
+        .student-levels
```
Comment (sderickson) [inline app/templates/teachers/teacher-student-view.jade]: Should this class be different? Or renamed to be more generic, seems like it refers to the above div, but this new div is just using the same styling.

File: app/templates/teachers/hovers/progress-dot-single-student-level.jade [hunk 2]
```diff
@@ -19,9 +19,10 @@ if completed
     if dateCompleted
       span= new Date(dateCompleted).toLocaleString()
     +timePlayed
-  //- .small-details
-  //-   i(data-i18n='teacher.click_to_view_solution')
-  //-     | click to view solution
+  if !isProject && !isLadder
+    .small-details.nowrap
+      //- TODO: data-i18n doesn't work here
```
Comment (sderickson) [inline app/templates/teachers/hovers/progress-dot-single-student-level.jade]: It doesn't? Need me to take a look at why?

Comment (differentmatt) [reply app/templates/teachers/hovers/progress-dot-single-student-level.jade]: Not a ship blocker, but you can if you want.  I've spent 20 minutes on this so far, and can't really justify any more at this point.

File: app/templates/teachers/teacher-student-view.jade (hunk 1 above)
Comment (differentmatt) [reply app/templates/teachers/teacher-student-view.jade]: Yea, it sure is.  Will rename.

**Pull request merged**